plugins/modules/rabbitmq_user.py

```diff
@@ -127,93 +127,220 @@
     state: present
 '''
 
+import distutils.version
+import json
+import re
+
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.common.collections import count
 
 
+def normalized_permissions(vhost_permission_list):
+    """Older versions of RabbitMQ output permissions with slightly different names.
+
+    In older versions of RabbitMQ, the names of the permissions had the `_priv` suffix, which was removed in versions
+    >= 3.7.6. For simplicity we only check the `configure` permission. If it's in the old format then all the other
+    ones will be wrong too.
+    """
+    for vhost_permission in vhost_permission_list:
+        if 'configure_priv' in vhost_permission:
+            yield {
+                'configure': vhost_permission['configure_priv'],
+                'read': vhost_permission['read_priv'],
+                'write': vhost_permission['write_priv'],
+                'vhost': vhost_permission['vhost']
+            }
+        else:
+            yield vhost_permission
+
+
+def as_permission_dict(vhost_permission_list):
+    return dict([(vhost_permission['vhost'], vhost_permission) for vhost_permission
+                 in normalized_permissions(vhost_permission_list)])
+
+
+def only(vhost, vhost_permissions):
+    return {vhost: vhost_permissions.get(vhost, {})}
+
+
+def first(iterable):
+    return next(iter(iterable))
+
+
 class RabbitMqUser(object):
     def __init__(self, module, username, password, tags, permissions,
                  node, bulk_permissions=False):
         self.module = module
         self.username = username
-        self.password = password
+        self.password = password or ''
```

Comment:
Is it possible for rabbitmq to allow an empty password? If so, this change may cause some problems.